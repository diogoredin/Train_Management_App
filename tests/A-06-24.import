SERVICE|390|12.2|06:05|A1|06:13|A211|06:14|B4|06:26|A411|06:48|A511|06:57|A611|07:01|A711|07:06|A811|07:10|A9
SERVICE|990|12.2|07:02|A1|07:03|A2|07:04|B3|07:26|A4|07:48|A5|07:57|A6|08:01|A7|08:06|A8|08:10|A9
SERVICE|490|12.2|06:02|A1|06:03|A21|06:04|B4|06:26|A41|06:48|A51|06:57|A61|07:01|A71|07:06|A81|07:10|A9
SERVICE|590|12.2|09:02|A1|09:03|A22|09:04|B5|09:26|A42|09:48|A52|09:57|A62|09:01|A72|10:06|A82|10:10|A9
SERVICE|690|12.2|07:06|B1|07:17|B2|07:31|B3|07:53|B41|08:14|B5|08:22|B6|08:25|B7
SERVICE|670|12.2|07:06|B1|07:17|B2|07:31|B31|07:53|B4|08:14|B51|08:24|B6|08:35|B7
SERVICE|780|12.2|09:06|B1|09:17|B2|09:31|B5|09:53|B41|10:14|B51|10:24|B6|10:38|B7
PASSENGER|Azura
PASSENGER|Boethiah
PASSENGER|Clavicus Vile
PASSENGER|Hermaeus Mora
PASSENGER|Hircine
PASSENGER|Malacath
PASSENGER|Mehrunes Dagon
PASSENGER|Mephala
PASSENGER|Meridia
