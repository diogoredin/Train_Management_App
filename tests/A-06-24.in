6
3
1
A1
B7
2017-12-31
06:00
0
0
0
