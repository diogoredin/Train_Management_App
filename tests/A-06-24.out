Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-31 @ 6.12
Serviço #490 @ 0.36
06:02 A1
06:03 A21
06:04 B4
Serviço #670 @ 5.76
07:53 B4
08:14 B51
08:24 B6
08:35 B7

Itinerário 2 para 2017-12-31 @ 7.45
Serviço #390 @ 1.69
06:05 A1
06:13 A211
06:14 B4
Serviço #670 @ 5.76
07:53 B4
08:14 B51
08:24 B6
08:35 B7

Itinerário 3 para 2017-12-31 @ 8.70
Serviço #990 @ 0.36
07:02 A1
07:03 A2
07:04 B3
Serviço #690 @ 8.34
07:31 B3
07:53 B41
08:14 B5
08:22 B6
08:25 B7

Itinerário 4 para 2017-12-31 @ 9.24
Serviço #590 @ 0.36
09:02 A1
09:03 A22
09:04 B5
Serviço #780 @ 8.88
09:31 B5
09:53 B41
10:14 B51
10:24 B6
10:38 B7
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 